active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,660 @@
+
+    static Windows.UI.Xaml.DependencyProperty TitleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SubtextProperty{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty ActionButtonTextProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ActionButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ActionButtonCommandProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ActionButtonCommandParameterProperty{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty CloseButtonTextProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty CloseButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty CloseButtonCommandProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty CloseButtonCommandParameterProperty{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty TargetOffsetProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty IsLightDismissEnabledProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ShouldConstrainToRootBoundsProperty{ get; }; 
+    static Windows.UI.Xaml.DependencyProperty PreferredPlacementProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty HeroPlacementProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty PointerModeProperty{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty HeroContentProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty IconSourceProperty{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty TemplateSettingsProperty{ get; };
+}
+```
+
+# Appendix
+
+### Visual Components
+
+ | Component |  Notes |
+|:---:|:---|
+| Container | * The container is the body of the tip and encapsulates all the tip components. <br> * Nonmodal. <br> * If content height exceeds max height or width, vertical scrolling will be enabled. See Scroll. <br> * For visibility concerns, the container has a border around the outer edge, which adheres to the pointer if present. See Pointer. <br> * For visibility concerns, the top edge of the container has a 1px highlight which also adheres to the pointer if present. See Pointer. <br><br> ![An empty teaching tip.](Container.PNG) |
+| Title| * Semi-bolded. <br> * Text wraps at Close (X) Button and Container border. <br><br> ![A teaching tip with a title and subtitle populated with sample text.](Title.PNG) |
+| Subtitle | * Text wraps at Close (X) Button and Container border. <br><br> ![A teaching tip with a subtitle that reads "Body text in a minimum height tip.](Subtitle.PNG) |
+| Content | * Can be customized to include text, images, videos, animation, checkboxes, hyperlinks, and any other XAML content. <br> * Will scroll if there is more content to show than tip height allows. See Scroll Bar. <br> * Placed below Subtitle and above Close/Action Button. <br><br> ![A teaching tip with a sample title and subtitle. The teaching tip has a picture of a mountain in the content area.](Content.PNG) |
```

Comment:
@MikeHillberg, this kind of description with embedded image is a knock against the new format of the docs being inline. Any thoughts around how to have the best of both worlds?

Reply:
My current thought is both formats are fine for the reader, so whatever's easier for the writer is a good answer. In particular here I didn't worry about trying to change the format anyway, since it had already been written.

Reply:
Is this refering to MSDN documentation format or our proposal/spec templates? Apologies - I think I am lacking some context for this comment.

Reply:
Spec template. The current process says to do what you've done, there's a proposal to put these descriptions into the IDL as comments. But for now I say let's leave this as you have it.